core/iwasm/compilation/aot_emit_function.c

```diff
@@ -882,11 +884,7 @@ aot_compile_op_call(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
 
     if (func_type->result_count > 0) {
         /* Push the first result to stack */
-        if (comp_ctx->enable_gc
-            && aot_is_type_gc_reftype(func_type->types[func_type->param_count]))
-            PUSH_GC_REF(value_ret);
-        else
-            PUSH(value_ret, func_type->types[func_type->param_count]);
+        PUSH(value_ret, func_type->types[func_type->param_count]);
```

Comment:
For GC ref types, if we don't use `PUSH_GC_REF`, can `aot_value_stack_push` process this type correctly?

Reply:
I converted gc ref types into VALUE_TYPE_GC_REF in PUSH macro, see changes in aot_compiler.h.

Reply:
Got it, thanks~